cli/cmd/dashboard.go

```diff
@@ -33,10 +37,26 @@ var dashboardCmd = &cobra.Command{
 			os.Exit(1)
 		}
 
+		client, err := newPublicAPIClient()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed to initialize API Client: %+v\n", err)
+		}
+
+		dashboardAvailable, err := isDashboardAvailable(client)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "Failed while checking availability of dashboard: %+v\n", err)
+		}
+
+		if !dashboardAvailable {
+			fmt.Fprint(os.Stderr, "Conduit dashboard might not be installed in your cluster\n")
```

Comment:
i liked what you had before... `Conduit dashboard is not installed in cluster`

Reply:
Since there's nothing called _dashboard_ in the controller, we may want to explicitly call out the `web` deployment or whatever the user could actually look at to diagnose this.